backend/reservations/models.py

```diff
@@ -6,6 +6,7 @@ class Guest(models.Model):
     ticket = models.CharField("Ticket", max_length=20)
     invitation = models.CharField("Invitation", max_length=20)
     jwt = models.CharField("JWT", max_length=240)
+    notes = models.TextField(blank=True, verbose_name='GuestNotes')
```

Comment:
Are these guest notes coming from the rooms spreadsheet? Or planned use?

Reply:
And invitation is coming from the guest list?

Reply:
Guest notes are from the rooms spreadsheet - part of why I was thinking we need to add "stub" guests. Invitation comes from the guest list.

Reply:
Dropped the notion of stub guests in favor of just throwing a name on the room. Duplication of data, but given the disconnect of who "owns" the room and who is "residing" in it this seems to fit.